content/application-faq/what-is-gold-card.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "What is the Employment Gold Card?"
```

Comment:
I think we need this information, but I'm not sure adding an entire section for one question was the best way to do it. As I mentioned in chat just before going to sleep, there are some ideas on how to do this that will make this bit more substantial and worthy of a navigation entry :)